src/signals/incident/components/form/MapSelectors/Asset/AssetList/AssetList.tsx

```diff
@@ -57,40 +64,35 @@ const AssetList: FunctionComponent<AssetListProps> = ({
     }
 
     return {
-      id,
-      label: `${description}${id ? ` - ${id}` : ''}`,
+      ...baseItem,
       iconUrl: icon ? `data:image/svg+xml;base64,${btoa(icon.iconSvg)}` : '',
       isReported: false,
     }
-  })
+  }, [featureTypes, selection])
 
   return (
     <IconList data-testid="assetList" className={className}>
-      {items.map((item) => (
-        <IconListItem
-          key={item.id}
-          id={
-            item.isReported
-              ? `assetListItem-${item.id}-reported`
-              : `assetListItem-${item.id}`
-          }
-          iconUrl={item.iconUrl}
-        >
-          <ItemWrapper>
-            {item.label}
-            {onRemove && (
-              <StyledButton
-                data-testid={`assetListRemove-${item.id}`}
-                aria-label="Verwijder"
-                icon={<Close />}
-                onClick={() => {
-                  onRemove(item.id.toString())
-                }}
-              />
-            )}
-          </ItemWrapper>
-        </IconListItem>
-      ))}
+      <IconListItem
+        key={item.id}
+        id={
+          item.isReported
+            ? `assetListItem-${item.id}-reported`
+            : `assetListItem-${item.id}`
+        }
+        iconUrl={item.iconUrl}
+      >
+        <ItemWrapper>
+          {item.label}
+          {onRemove && (
+            <StyledButton
+              data-testid={`assetListRemove-${item.id}`}
+              aria-label="Verwijder"
+              icon={<Close />}
+              onClick={onRemove}
+            />
+          )}
+        </ItemWrapper>
+      </IconListItem>
     </IconList>
   )
```

Comment:
If it is only possible to have one item here, shouldn't the list be removed?

Reply:
We can, but the list component has styling for the icons and it was easier to leave it as it is.